weave-js/tailwind.config.cjs

```diff
@@ -3,6 +3,47 @@ module.exports = {
   content: ['./src/**/*.{ts,tsx}'],
   darkMode: ['class', '[data-mode="dark"]'],
   theme: {
+    /**
+     * This overrides Tailwind's default spacing system.
+     * Inherited by padding, margin, width, height, maxHeight, gap,
+     * inset, space, translate, scrollMargin, and scrollPadding
+     */
+    spacing: {
+      0: '0rem',
+      px: '0.063rem',
+      1: '0.063rem',
+      2: '0.125rem',
+      3: '0.188rem',
+      4: '0.25rem',
+      5: '0.313rem',
+      6: '0.375rem',
+      7: '0.438rem',
+      8: '0.5rem',
+      10: '0.625rem',
+      12: '0.75rem',
+      14: '0.875rem',
+      16: '1rem',
+      18: '1.125rem',
+      20: '1.25rem',
+      22: '1.375rem',
+      24: '1.5rem',
+      26: '1.625rem',
+      28: '1.75rem',
+      30: '1.875rem',
+      32: '2rem',
+      34: '2.125rem',
+      36: '2.25rem',
+      38: '2.375rem',
+      40: '2.5rem',
+      44: '2.75rem',
+      48: '3rem',
+      56: '3.5rem',
+      60: '3.75rem',
+      64: '4rem',
+      72: '4.5rem',
+      80: '5rem',
+      96: '6rem',
+    },
```

Comment:
**question:** what do we do for values over 96px? particularly for height & width may need much larger values

Reply:
That will probably be added in a future PR - we'd need to design with designers about this